### PR TITLE
Bugfix/issue 98 lkj corr cholesky fvar

### PR DESCRIPTION
**Pull request by rtrangucci**
#### Summary:

Allows for fvar instantiation of lkj_corr_cholesky.

#### Intended Effect:

Admits higher-order autodiff of lkj_corr_cholesky density

#### How to Verify:

./runTests.py test/unit/math/mix/mat/prob/lkj_corr_test.cpp

#### Side Effects:

None.

#### Documentation:

None.

#### Reviewer Suggestions:

Anyone

### CODE REVIEW

File: test/unit/math/rev/mat/prob/test_gradients.hpp [hunk 1]
```diff
@@ -2,6 +2,12 @@
 #include <vector>
 #include <iomanip>
 
+void test_grad_eq(Eigen::Matrix<double, -1, 1> grad_1,
+                    Eigen::Matrix<double, -1, 1> grad_2) {
+  for (int i = 0; i < grad_1.size(); ++i)
```
Comment (syclik) [inline test/unit/math/rev/mat/prob/test_gradients.hpp]: @rtrangucci, add a quick check for size grad_1 == size grad_2 if you want to reuse this function.

**Comment by rtrangucci** [comment]
What's the deal here? It looks like everything passed...


Rob

**Comment by syclik** [comment]
I'll take a look. I might have done something weird.

**Comment by syclik** [comment]
Yeah, looks like it passed just fine.

**Pull request merged**